apps/maxkb/1.8.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   maxkb:
-    image: 1panel/maxkb:v1.8.0
+    image: 1panel/maxkb:v1.8.1
     container_name: ${CONTAINER_NAME}
     restart: unless-stopped
     ports:
```

Comment:
There is no apparent issue with the given YAML snippet regarding irregularities, potential issues, or optimization opportunities at this time. The difference between the two images (v1.8.0 to v1.8.1) indicates that it's an update to fix any bugs or add new features in the latest version of MaxKB. If you require further updates or need help configuring your service, feel free to ask!